src/platform/packages/shared/kbn-esql-ast/src/commands_registry/commands/fork/validate.ts

```diff
@@ -68,9 +68,10 @@ export const validate = (
     messages.push(errors.tooManyForks(forks[1]));
   }
 
-  context?.fields.set('_fork', {
+  context?.columns.set('_fork', {
```

Comment:
Can we also remove this @drewdaemon ?

Reply:
Gone in https://github.com/elastic/kibana/pull/233221/commits/383cdbd51c1978856406ea0f637426273087a2c4